src/launch-context.ts

```diff
@@ -67,6 +67,9 @@ export class LaunchContext<
 
     private readonly _reservedFieldNames = [...Reflect.ownKeys(this), 'extend'];
 
+    // TODO: change this to PropertyKey
```

Comment:
```suggestion
    // TODO: change this to PropertyKey when TypeScript 4.4 releases
```

https://github.com/microsoft/TypeScript/pull/44512 :)